transport/tchannel/channel_outbound_test.go

```diff
@@ -155,95 +155,139 @@ func TestChannelOutboundHeaders(t *testing.T) {
 }
 
 func TestChannelCallSuccess(t *testing.T) {
-	server := testutils.NewServer(t, nil)
-	defer server.Close()
-	serverHostPort := server.PeerInfo().HostPort
-
-	server.GetSubChannel("service").SetHandler(tchannel.HandlerFunc(
-		func(ctx context.Context, call *tchannel.InboundCall) {
-			assert.Equal(t, "caller", call.CallerName())
-			assert.Equal(t, "service", call.ServiceName())
-			assert.Equal(t, tchannel.Raw, call.Format())
-			assert.Equal(t, "hello", call.MethodString())
+	tests := []struct {
+		msg        string
+		addSvcName bool
+	}{
+		{
+			msg:        "test channel call success with service name response header",
```

Comment:
Let's make these messages unique, so that if one of them fails we can tell which one failed.

Reply:
Resolved.